scripts/depcheck.ts

```diff
@@ -30,12 +30,15 @@ const fix: boolean = cli.opts()['fix'];
 const fixTS = cli.opts()['fixTsconfig'];
 const verbose = cli.opts()['verbose'];
 
+const testFilePatterns = [
+  // files matching these patterns will be ignored
+  /.*.spec.tsx?/,
+  /.*.story.tsx?/,
+  /.*.example.tsx?/,
+  /.*.testutils.tsx?/,
+];
```

Comment:
`/.*stories.tsx` should be here. That should be the new standard we build towards. I mention this in the folder structure guidelines PR as well, but that's the standard that Storybook documents.